shared/build.gradle.kts

```diff
@@ -126,4 +123,8 @@ android {
         jvmToolchain(11)
     }
 }
+dependencies {
+    testImplementation("org.junit.jupiter:junit-jupiter:5.8.1")
+    testImplementation("org.junit.jupiter:junit-jupiter:5.8.1")
+}
```

Comment:
duplication? why is it not in the library catalog?
and it may be in
```
androidUnitTest.dependencies {
    implementation("org.junit.jupiter:junit-jupiter:5.8.1")
}
```

Reply:
sorry, it was android studio mess, will drop